<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color: rgba(255,255, 255, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color: rgba(255,255, 255, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Hilfe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color: rgba(255,255, 255, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Feedback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-style: solid; border-radius: 10px; background-color:rgba(26, 97, 168, 0.55); color:white;}
QPushButton:Hover{  background-color: rgba(255,255, 255, 0.55); color: black; border: 2px solid rgba(26, 97, 168, 0.99);}</string>
    </property>
    <property name="text">
     <string>Einstellung</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
